pkg/logentry/stages/match_test.go

```diff
@@ -57,45 +53,6 @@ var testMatchLogLineApp2 = `
 }
 `
 
-func TestMatchPipeline(t *testing.T) {
```

Comment:
This was removed because it doesn't make sense to try to get a pipeline duration with channels: you might never receive and output or you might receive multiple outputs for the same line.